src/visualisation/Visual.cpp

```diff
@@ -10,30 +10,28 @@
 #include <utility>
 
 #include "Visual.h"
-#include "../utility/Parameters.h"
 #include "../Domain.h"
 #include "CSVWriter.h"
 #include "VTKWriter.h"
 
-Visual::Visual(const Solution &solution, bool has_analytical_solution) :
+Visual::Visual(Settings const &settings, Solution const &solution, bool has_analytical_solution) :
         m_solution(solution),
         m_has_analytical_solution(has_analytical_solution) {
 #ifndef BENCHMARKING
-    m_logger = Utility::create_logger(typeid(this).name());
+    m_logger = Utility::create_logger(settings, typeid(this).name());
 #endif
-    auto params = Parameters::getInstance();
-    m_filename = Utility::remove_extension(params->get_filename());
+    m_filename = Utility::remove_extension(settings.get_filename());
 
-    m_save_csv = (params->get("visualisation/save_csv") == XML_TRUE);
-    m_save_vtk = (params->get("visualisation/save_vtk") == XML_TRUE);
+    m_save_csv = settings.get_bool("visualisation/save_csv");
+    m_save_vtk = settings.get_bool("visualisation/save_vtk");
 
-    m_dt = params->get_real("physical_parameters/dt");
-    m_t_end = params->get_real("physical_parameters/t_end");
+    m_dt = settings.get_real("physical_parameters/dt");
```

Comment:
remove member variable `m_nu` instead access value through Settings

Reply:
done in 0d1d248
took (m_dt, m_t_end). Guess it was a copy past typo.